docs/usage.rst

```diff
@@ -262,6 +270,19 @@ method
 follow_redirects
   (*Optional*) Whether to follow 3xx HTTP redirects. Default is ``false``. If you are looking at a ``Location`` header, you shouldn't change this.
 
+Search with an HTML Parser
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+::
+
+  source = "htmlparser"
+
+Send an HTTP request and search through the body a specific xpath.
```

Comment:
We need to note that an additional dependency `lxml` is required.

Reply:
Should I add the dependency in the setup.py "install_requires" ?

Reply:
No, because only users who use this source plugin need it, and lxml is quite large.